### PR TITLE
Tasks 87718 90353

### PR DESCRIPTION
**Pull request by suskiwojciech**

### CODE REVIEW

File: src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            fakedContext = new XrmFakedContext();
+            pluginContext = fakedContext.GetDefaultPluginContext();
+        }
+
+        [TestMethod]
+        public void Test_1()
```
Comment (Antek012) [inline src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs:29]: fix name to match convention

File: src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            {
+                { invln_gettypeofhomeslistRequest.Fields.invln_applicationid, appId.ToString()},
+                { invln_gettypeofhomeslistRequest.Fields.invln_organisationid, organizationId.ToString()},
+                { invln_gettypeofhomeslistRequest.Fields.invln_userid, userId.ToString()},
+                { invln_gettypeofhomeslistRequest.Fields.invln_fieldstoretrieve, null}
+            };
+
+            fakedContext.ExecutePluginWithConfigurations<GetTypeOfHomesListPlugin>(pluginContext, "", "");
+            var outputResponce = pluginContext.OutputParameters[invln_gettypeofhomeslistResponse.Fields.invln_hometypeslist].ToString();
+            Assert.IsNotNull(outputResponce);
```
Comment (Antek012) [inline src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs:78]: If you want to add tests you should assert if collection is equivalent
now you are checking if collection is not empty and not null

File: src/HE.Common.IntegrationModel/PortalIntegrationModel/HomeTypeDto.cs [hunk 3]
```diff
@@ -47,5 +48,9 @@ public class HomeTypeDto
         public bool? targetRentOver80PercentOfMarketRent { get; set; }
         public decimal? expectedFirstTrancheSaleReceipt { get; set; }
         public decimal? proposedRentAsPercentOfUnsoldShare { get; set; }
+        public bool? mmcApplied { get; set; }
+        public List<int> mmcCategories { get; set; }
+        public List<int> mmcCategories_1subcategories { get; set; }
```
Comment (Antek012) [inline src/HE.Common.IntegrationModel/PortalIntegrationModel/HomeTypeDto.cs:53]: Is this a valid naming convention?
I think we are not using underscore

**Review by Antek012 — APPROVED** [review]

File: src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs (hunk 1 above)
Comment (suskiwojciech) [reply src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs]: fix

File: src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs (hunk 2 above)
Comment (suskiwojciech) [reply src/HE.CRM.AHP.Plugins.Tests/CustomApi/GetTypeOfHomesListPluginTests.cs:81]: Fixed

**Comment by suskiwojciech** [comment]
Problem wit tests

**Pull request closed without merging**